docs/WALKTHROUGH.md

```diff
@@ -0,0 +1,394 @@
+brokerName: ups-broker
+osbGuid: 4F6E6CF6-FFDD-425F-A2C7-3C9258AD2468
+bindable: false
+planUpdatable: false
+plans:
+- name: default
+  osbFree: true
+  osbGuid: 86064792-7ea2-467b-af93-ac9694d96d52
+```
+
+## Step 6 - Provisioning a new Instance
+
+Now that one or more `ServiceClass` resources are in the catalog, we can provision a new
+instance of the `user-provided-service`. We do this by creating a new 
+[`Instance`](../contrib/examples/walkthrough/ups-instance.yaml) resource for each provision:
+
+```console
+./kubectl16 create -f contrib/examples/walkthrough/ups-instance.yaml
+```
+
+That operation should output:
+
+```console
+instance "ups-instance" created
+```
+
+Now that the new `Instance` is created, we can check the status of it with:
+
+```console
+kubectl get instances -n test-ns ups-instance -o yaml
+```
+
+We should see something like:
+
+```yaml
+apiVersion: servicecatalog.k8s.io/v1alpha1
+kind: Instance
+metadata:
+  creationTimestamp: 2017-03-03T04:26:08Z
+  name: ups-instance
+  namespace: test-ns
+  resourceVersion: "9"
+  selfLink: /apis/servicecatalog.k8s.io/v1alpha1/namespaces/test-ns/instances/ups-instance
+  uid: 8654e626-ffc9-11e6-b111-0242ac110005
+spec:
+  osbGuid: 34c984e1-4626-4574-8a95-9e500d0d48d3
+  planName: default
+  serviceClassName: user-provided-service
+status:
+  conditions:
+  - message: The instance was provisioned successfully
+    reason: ProvisionedSuccessfully
+    status: "True"
+    type: Ready
+```
+
+## Step 7 - Bind to the Instance
+
+Now that our `Instance` has been created, we can bind to it. After the bind operation is
+complete on the UPS broker server, the service catalog will write the resulting credentials 
+to a Kubernetes secret. Here's how we create the new 
+[`Binding`](../contrib/examples/walkthrough/ups-binding.yaml):
+
+
+```console
+kubectl create -f contrib/examples/walkthrough/ups-binding.yaml
+```
+
+That command should output:
+
+```console
+binding "ups-binding" created
+```
+
+We can check the status of the `Binding` using `kubectl get`:
+
+```console
+kubectl get bindings -n test-ns ups-binding -o yaml
+```
+
+We should see something like:
+
+```yaml
+apiVersion: servicecatalog.k8s.io/v1alpha1
+kind: Binding
+metadata:
+  creationTimestamp: 2017-03-07T01:44:36Z
+  finalizers:
+  - kubernetes
+  name: ups-binding
+  namespace: test-ns
+  resourceVersion: "29"
+  selfLink: /apis/servicecatalog.k8s.io/v1alpha1/namespaces/test-ns/bindings/ups-binding
+  uid: 9eb2cdce-02d7-11e7-8edb-0242ac110005
+spec:
+  instanceRef:
+    name: ups-instance
+  osbGuid: b041db94-a5a0-41a2-87ae-1025ba760918
+  secretName: my-secret
+status:
+  conditions:
+  - message: Injected bind result
+    reason: InjectedBindResult
+    status: "True"
+    type: Ready
+```
+
+Notice that the status has a ready condition set.  This means our binding is
+ready to use.  If we look at the secrets in our `test-ns` namespace in
+kubernetes, we should see:
+
+```console
```

Comment:
I think that's what the implication "in Kubernetes" is, but it would be nice to illustrate for each kubectl command what context should be used.